src/logic/safe/store/models/types/gateway.d.ts

```diff
@@ -34,13 +33,13 @@ type TxHistoryLocation = 'history'
 
 export type TxLocation = TxHistoryLocation | TxQueuedLocation
 
-type HistoryGatewayResult = GatewayDefinitions['TransactionListItem']
+type HistoryGatewayResult = TransactionListItem
```

Comment:
Do these now use the assigned types? i.e.
```ts
type GatewayDefinitions = {
  TractionListItem: TransactionListItem
  ...
}
```

Reply:
There are a few places like `This['type']` to `Type` as well.

Reply:
I didn't get the question.
I grepped for `GatewayDefinitions` and replaced all the occurrences. Did I miss anything?